src/OmniSharp.Abstractions/IsExternalInit.cs

```diff
@@ -0,0 +1,4 @@
+﻿namespace System.Runtime.CompilerServices
+{
+    internal sealed class IsExternalInit { }
```

Comment:
I assume this is so that records compile? maybe it should also have `[EditorBrowsable(EditorBrowsableState.Never)]`

Reply:
Yeah, it's for records. I can certainly add that attribute.